docs/extensions/tutorials/_posts/1970-01-01-SettingLocalEnvironment.md

```diff
@@ -21,114 +21,95 @@ Before setting up a local environment for building Shoutem extensions, set up Re
-
+Save the changes. Now you don't need to do `shoutem push` and wait for the Builder to load the changes, you can just reload the app:
 - On simulator:
   - iOS: press `Command ⌘` + `R`
   - Android: press `R` twice
-- On physical device: shake your device to open up **Developer Menu** and select **Reload**
+- Using `shoutem run`: shake your device to open up **Developer Menu** and select **Reload**
 
 If you enable automatic reloading, the previous step is unnecessary. Having local environment set, you can also debug your extension. Follow the React Native guide on [debugging](https://facebook.github.io/react-native/docs/debugging.html) where you can find out how automatic reloading works as well.
 
-To see which extensions are linked, as well as which developer is signed in, run:
+### Managing your app's Configuration
 
-```ShellSession
-$ shoutem show
-Linked directories:
-  /Path/to/hello-world
-Registered as `{{ site.example.devName }}`.
-```
+This section will cover the uses of the configuration script and associated commands.
 
-To unlink an extension, locate to that folder and do:
+#### shoutem configure
 
 ```ShellSession
-$ shoutem unlink
-Unlink successful. Please, kill the packager before running the app.
+$ shoutem configure
+...
 ```
 
-... or just delete the extension.
+You use it when:
+    - creating and installing a new extension locally using `shoutem init`, `shoutem push` and `shoutem install`
+    - making changes (adding, removing, etc.) to dependencies of the app
+    - making changes to the app's build script
 
-You can also do `shoutem unlink --all`, but make sure you don't mind unlinking all the extensions you have linked.
-
-## Getting the Full App Code
-
-When you push your extension to Shoutem, we build the app with that extension for you. However, sometimes you might want to see the full picture - what the complete app project looks like. To get your Shoutem-built app's code, you can use:
+#### shoutem configure --release
 
 ```ShellSession
-$ shoutem pull-app
-Select the app you want to pull: News app ({{ site.example.appId }})
-Pulling the app `News app`...
-Pulling extensions...
-Success!
-Change your working directory to `News_app`.
-```
-
-Locate to `News_app` directory and open it:
-
-```ShellSession
-$ cd News_app
-$ open .
+$ shoutem configure --release
+...
 ```
 
-Check what the project structure looks like. It should look familiar, because this is what regular React Native projects look like. The `index.js` file is the starting point. The `extensions` folder contains all the extensions installed in the app (all of the ones in the **Extensions** tab of your app in the Builder, not just the ones whose Screens you've added).
+You use it when preparing your app for release. This prepares the Shoutem app for the building process regular React Native apps go through when preparing for release.
```

Comment:
`configure --release` sets the `release` flag in the config, making certain extensions behave differently. This is why we say it `prepares the app for release`.

Source: Ivan Vuković

Ask him to confirm and/or update. I'm just documenting the way I've been informed.